.ruby-version

```diff
@@ -0,0 +1 @@
+3.4.0-preview2
```

Comment:
Let's do `3.4.1` as `preview2` must be our internal version which is not helpful for an open-source gem